lte/gateway/c/core/oai/tasks/amf/Registration.cpp

```diff
@@ -772,11 +760,9 @@ int amf_proc_registration_complete(amf_context_t* amf_ctx) {
   } else {
     OAILOG_WARNING(
         LOG_NAS_AMF,
-        "UE Context not found for "
+        "ue context not found for "
```

Comment:
```suggestion
        "UE context not found for "
```

Reply:
addressed comment, changed as you suggested.